### ISSUE TITLE
TensorFlow GPU, CUDA_ERROR_LAUNCH_FAILED on tf.one_hot()

### ISSUE THREAD

**Issue by luzi82** [tensorflow/tensorflow#6783]
Use tf.one_hot() on Windows 10, GPU, Nvidia 970.  Error CUDA_ERROR_LAUNCH_FAILED happens.

### What related GitHub issues or StackOverflow threads have you found by searching the web for your problem?

I have post an issue in http://stackoverflow.com/questions/41115476/tensorflow-gpu-cuda-error-launch-failed-on-tf-one-hot .  No solution found.  More people report the same issue.

### Environment info

- TensorFlow 0.12.0-rc1
- Python 3.5
- CUDA 8.0
- cuDNN 5.1
- OS: Windows 10
- GPU: GeForce GTX 970

### If possible, provide a minimal reproducible example (We usually don't have time to read hundreds of lines of your code)

```
import tensorflow as tf
idx_0 = tf.placeholder(tf.int64, [None])
mask = tf.one_hot(idx_0, 3, axis=-1)
sess = tf.Session()
sess.run(tf.global_variables_initializer())
a = sess.run([mask],feed_dict={idx_0:[0,1,2]})
print(a)
```

### What other attempted solutions have you tried?

1. Run the code in same machine, Linux, GPU.  No error.
2. Run the code in same machine, Windows, CPU.  No error.

### Logs or other output that would be helpful
(If logs are large, please upload as attachment or provide link).

```
E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_driver.cc:1177] could not synchronize on CUDA context: CUDA_ERROR_LAUNCH_FAILED :: No stack trace available
E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED
F c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_util.cc:370] GPU sync failed
```

**Comment by Carmezim** [comment]
Do you think it can be related to [GTX 970 memory issues](http://wccftech.com/nvidia-geforce-gtx-970-memory-issue-fully-explained/) when trying to allocate more than 3.5GB?

**Comment by Carmezim** [comment]
There is an issue with memory allocation on GTX 970 when trying to go over 3.5GB but in this case it seems related to #6509

**Comment by jart** [comment]
@mrry do you think this should be closed out as a duplicate of #6509?

**Comment by mrry** [comment]
Yes, let's close this as a duplicate, since it looks like it's the same problem (i.e. `tf.one_hot()` is broken for Windows/GPU).
